server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -278,4 +286,38 @@ export class TenantManager {
 
         await collection.update({ _id: tenantId }, { disabled: true }, null);
     }
+
+    private encryptAccessInfo(externalStorageData :any): any {
```

Comment:
Can we not use IExternalStorage instead of any?

Reply:
IExternalStorage is defined in @fluid-internal/frs-shared, which is not shared to the FFX